setup.py

```diff
@@ -80,8 +80,8 @@ def get_outputs(self):
     scripts=[],
     license='GPLv2',
     description='A fast alternative to freezegun that wraps libfaketime.',
-    long_description=(open('README.rst').read() + '\n\n' +
-                      open('CHANGELOG.rst').read()),
+    long_description=(open('README.md').read() + '\n\n' +
+                      open('CHANGELOG.md').read()),
```

Comment:
should we set [long_description_content_type](https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/) too?

Reply:
Here it is!